src/JWT.Extensions.AspNetCore/JwtAuthenticationEvents.cs

```diff
@@ -0,0 +1,100 @@
+namespace JWT.Extensions.AspNetCore
+{
+    using System;
```

Comment:
Can you move the uisnfa outside the namespace?

Reply:
I'll do a whole formatting pass at the end. :) my IDE is just putting things in my normal spots. But before we sign this puppy off we can square all of the formatting away. No problem. :)